imports/api/active-period.js

```diff
@@ -3,21 +3,22 @@ import { SimpleSchema } from 'meteor/aldeed:simple-schema';
 const TimePeriodSchema = new SimpleSchema({
   begin: { type: Date, optional: true,
     custom() {
+      const nowDate = new Date();
       const beginDate = this.value;
       const endDate = this.field('activeTime.end').value;
-      const nowDate = new Date();
       if (!beginDate && endDate) return 'required';
       if (beginDate && beginDate > nowDate) return 'notAllowed';
       return undefined;
     },
   },
   end: { type: Date, optional: true,
     custom() {
+      const nowDate = new Date();
       const beginDate = this.field('activeTime.begin').value;
       const endDate = this.value;
-      const nowDate = new Date();
+      if (endDate && !beginDate) return 'notAllowed';
```

Comment:
Az activeTime-ot mar megcsinaltuk a mater-en, most szerintem ne jojjenek be azzal kapcsolatban uj valtozasok. Csak a flow router-rel kapcsolatosak.
Peldaul ezt az esetet eddig a begin mezo bepirositasval jeleztuk. Az szerintem jo is volt ugy. Az hogy megadott end date-et, de nem adott meg begin-t. Nem az end a problemas, hanem hogy nincs begin.

Reply:
Nem egészen értem. A masteren történt a flow router, meg az active time módosítás is. Az assureFields eltávolításával csak az előző állapotot állítottam vissza, ami a masteren volt. Ez nem új változás, hanem visszaállítás az assureFields-et megelőző állapotra.

Reply:
Ez nem új változtatás, csak visszaálltam az assureFields előtti állapotra. Erről beszéltünk is. Nem tudtam kettészedni két külön PR-re, mert még nem lett elfogadva az előtő PR, ezért ráment az egész 1db-ra.

Reply:
Nézzem meg, hogy mi a jelenlegi állapot a masteren, írjam azzal felül és küldjek új commitot?

Reply:
Hat igen, most mar lehet hogy csak ugy lehet.
A jovoben ezt ugy lehet hatekonyabban, hogy 
git revert commit1 commit2 commit3
visszacsinalja azt a 3 commit-ot

Reply:
Addig hegeszd, amig ranezve erre a PR-re
https://github.com/edemo/Lidehouse/pull/217
azt latod, hogy csak azok a valtozasok vannak benne, amik szuksegesek most.